prow/jobs/kyma/components/event-bus/event-bus.yaml

```diff
@@ -30,20 +30,36 @@ job_labels_template: &job_labels_template
 
 presubmits: # runs on PRs
   kyma-project/kyma:
-    - name: kyma-components-event-bus
-      branches:
+    - branches:
         - master
       <<: *job_template
+      run_if_changed: "^components/event-bus/"
+      extra_refs:
+        - <<: *test_infra_ref
+          base_ref: master
       labels:
         <<: *job_labels_template
         preset-build-pr: "true"
+    - branches:
+        - release-0.6
+      <<: *job_template
+      always_run: true
+      extra_refs:
+        - <<: *test_infra_ref
+          base_ref: release-0.6
+      labels:
+        <<: *job_labels_template
+        preset-build-release: "true"
 
 postsubmits:
   kyma-project/kyma:
-    - name: kyma-components-event-bus
-      branches:
+    - branches:
         - master
```

Comment:
`-master` should be at the same level of indentation as `<<: *job_template`. Although it seems to be working OK, it can be misleading to others.